engine/src/main/java/org/camunda/bpm/engine/impl/errorcode/BuiltinExceptionCode.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information regarding copyright
+ * ownership. Camunda licenses this file to you under the Apache License,
+ * Version 2.0; you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.camunda.bpm.engine.impl.errorcode;
+
+import org.camunda.bpm.engine.CrdbTransactionRetryException;
+import org.camunda.bpm.engine.OptimisticLockingException;
+import org.camunda.bpm.engine.ProcessEngineException;
+
+/**
+ * The set of built-in exception codes the built-in {@link ExceptionCodeProvider}
+ * uses to assign a code to a {@link ProcessEngineException}.
+ */
+public enum BuiltinExceptionCode {
+
+  /**
+   * The default code assigned to a {@link ProcessEngineException} when no other code is assigned.
+   */
+  DEFAULT(0),
```

Comment:
```suggestion
  FALLBACK(0),
```
Is fallback error a better name?